connector/ldap/ldap.go

```diff
@@ -177,6 +178,7 @@ func (c *Config) openConnector(logger logrus.FieldLogger) (*ldapConnector, error
 		if field.val == "" {
 			return nil, fmt.Errorf("ldap: missing required field %q", field.name)
 		}
+		logger.Infof("ldap %s: %s", field.name, field.val)
```

Comment:
This is redundant. We should only call out if the user has missed a required field. Please remove this info message.